### PR TITLE
Releases docs updates

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description

From D&R:
* Remove open preview from titles but keep banners
* Update banners to specify charges for releases will start March 18th
* Check we use "releases" as the umbrella term for feature not "release environment"
* Check steps and screenshots are correct now the UI is updated
* Remove release envionment details in the UI from the coming soon section
* project ID is now required

Identified missing info:
* Add tour of the releases UI dashboard section to overview
* Add how to for viewing releases and commands for envs and components

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/release/releases-overview.adoc [hunk 1]
```diff
@@ -12,12 +12,12 @@ contentTags:
 
 NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available.
 
-Visualise and control your deployments with CircleCI release environments. The CircleCI releases feature currently supports deployments to Kubernetes, and Amazon SageMaker.
+Visualise and control your deployments with CircleCI releases. Deployments to Kubernetes and Amazon SageMaker are supported.
```
Comment (alexazl) [inline jekyll/_cci2/release/releases-overview.adoc:15]: Can we capitalize "releases" (i.e., `CircleCI Releases`)?

File: jekyll/_cci2/release/releases-overview.adoc [hunk 2]
```diff
@@ -12,12 +12,12 @@ contentTags:
...
 
 [#introduction]
 == Introduction
 
-CircleCI release environments give you visibility into your Kubernetes deployments in the CircleCI web app, as well as access to commands for release management. From the CircleCI web app, you can:
+CircleCI releases gives you visibility into your Kubernetes deployments within the CircleCI web app, as well as access to commands and visualisations for release management. From the CircleCI web app, you can:
```
Comment (alexazl) [inline jekyll/_cci2/release/releases-overview.adoc:20]: Same here - let's make sure we capitalize `CircleCI Releases`.

File: jekyll/_cci2/release/releases-overview.adoc [hunk 3]
```diff
@@ -51,13 +51,124 @@ Configure a release environment to install the CircleCI release agent into your
 
 Developers can monitor releases in the CircleCI web app, and run commands to manage those releases from the UI. Commands are monitored by CircleCI and relayed to the release agent, which then runs the commands for you.
 
+[#releases-concepts]
+== Concepts
+
+The following sections describe concepts that will help you fully understand how CircleCI releases integration and capabilities.
```
Comment (alexazl) [inline jekyll/_cci2/release/releases-overview.adoc:57]: This sentence seems off. Maybe it should be `fully understand CircleCI Releases integration and capabilities.`

File: jekyll/_cci2/release/set-up-a-release-environment.adoc [hunk 4]
```diff
@@ -12,7 +12,7 @@ contentTags:
 
 NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available.
 
-Visualise and control deployments to your Kubernetes cluster with CircleCI release environments. The steps outlined on this page guide you to set up a release environment and install the CircleCI release agent into your Kubernetes cluster.
+Visualise and control deployments to your Kubernetes cluster with CircleCIs. The steps outlined on this page guide you to set up a release environment and install the CircleCI release agent into your Kubernetes cluster.
```
Comment (alexazl) [inline jekyll/_cci2/release/set-up-a-release-environment.adoc:15]: Replace `CircleCIs` with `CircleCI Releases`.

**Review by imlogang — APPROVED** [review]

**Review by alexazl — APPROVED** [review]

**Pull request merged**